README.md

```diff
@@ -1,21 +1,36 @@
 
-**The DuckDuckHack community and DuckDuckGo staff are working together on a mission to create the best search engine for programmers. To fully focus there, we are only accepting Pull Requests and Issues related to the [Programming Mission](https://forum.duckduckhack.com/t/duckduckhack-programming-mission-overview/53).**
+### The Programming Mission
+The current focus of DuckDuckHack is to create the best search engine for programmers. For now, we are **only accepting Pull Requests and Issues related to the Programming Mission**.
 
-[**Join the Programming Mission!**](https://forum.duckduckhack.com/t/duckduckhack-programming-mission-overview/53)  
-*Your help and expertise can have a huge impact on improving the search experience for developers and DuckDuckGo users around the world!*
 
-More Resources:   
-*[Full documentation](http://docs.duckduckhack.com)*  
-*[Instant Answers in Production](https://duck.co/ia)*  
-*[Instant Answers in Beta](https://beta.duckduckgo.com/?q=test&t=hc&ia=iatesting&iax=1)*  
+## Get Started
+Your help and expertise can improve the search experience for millions of DuckDuckGo users around the world!
 
-Instant Answers appear at the top of search results...
-![](http://docs.duckduckhack.com/assets/python_apply_at_DuckDuckGo.png)
+There are two ways you can get involved:
 
-Attribution is shown for developers who create or improve them...
-![](http://docs.duckduckhack.com/assets/attribution_example_1.png)
+1. [**Create new Fatheads, and improve existing ones**](https://github.com/duckduckgo/zeroclickinfo-fathead/issues?q=is%3Aopen+is%3Aissue+label%3A"Mission%3A+Programming")
+    - **Note**: This will require knowledge of Perl, Python, Ruby, or JavaScript (Node.js)
+2. [**Visit DuckDuckHack.com**](https://duckduckhack.com) to learn more about the Programming Mission, and how you can help us **analyze Instant Answer performance data** to determine new projects
+
+
+### What are Fathead Instant Answers?
+A set of key-value pairs, stored in a database. Fatheads provide unique answers (values) to a specific set of queries (keys). The key-value pairs are generated by parsing static data sources, such as technical documentation.
+
+#### Example
+
+- CSS Fathead
+    - [Code](https://github.com/duckduckgo/zeroclickinfo-fathead/tree/master/lib/fathead/mdn_css) | [Example Query](https://duckduckgo.com/?q=css+background-position&ia=about) | [Instant Answer Page](https://duck.co/ia/view/mdn_css)
+
+**Here's how it looks!**
+
+![css background-position search](https://cloud.githubusercontent.com/assets/873785/19787980/cdf51566-9c73-11e6-9ef2-ac44dede62c7.png)
+
+
+## Other Resources
+
+- [Fathead technical documentation](https://docs.duckduckhack.com/resources/fathead-overview.html)
+- [DuckDuckHack Slack](https://quackslack.herokuapp.com/)
```

Comment:
Include a forum link here?
And perhaps a link to live programming fatheads: https://duck.co/ia?repo=fathead&topic=programming